RedHat-UX/red-hat-design-system
# fix: lib helper elements

## What I did

1. move context-provider to `/lib/elements`
2. keep transparent deprecated entrypoint at `/elements/rh-context-provider/rh-context-provider.js`
3. add rh-context-picker at `/lib/elements`

## Testing Instructions

1. check context demos
- [rh-accordion/demo/](https://deploy-preview-996--red-hat-design-system.netlify.app/elements/accordion/demo/)
- [rh-accordion/demo/color-context/](https://deploy-preview-996--red-hat-design-system.netlify.app/elements/accordion/demo/color-context/)
- [rh-alert/demo/](https://deploy-preview-996--red-hat-design-system.netlify.app/elements/alert/demo/)
- [rh-avatar/demo/](https://deploy-preview-996--red-hat-design-system.netlify.app/elements/avatar/demo/)
- [rh-avatar/demo/color-context/](https://deploy-preview-996--red-hat-design-system.netlify.app/elements/avatar/demo/color-context/)
- [rh-blockquote/demo/](https://deploy-preview-996--red-hat-design-system.netlify.app/elements/blockquote/demo/)
- [rh-button/demo/](https://deploy-preview-996--red-hat-design-system.netlify.app/elements/button/demo/)
- [rh-card/demo/](https://deploy-preview-996--red-hat-design-system.netlify.app/elements/card/demo/)
- [rh-card/demo/color-context/](https://deploy-preview-996--red-hat-design-system.netlify.app/elements/card/demo/color-context/)
- [rh-cta/demo/](https://deploy-preview-996--red-hat-design-system.netlify.app/elements/call-to-action/demo/)
- [rh-cta/demo/color-context/](https://deploy-preview-996--red-hat-design-system.netlify.app/elements/call-to-action/demo/color-context/)
- [rh-spinner/demo/](https://deploy-preview-996--red-hat-design-system.netlify.app/elements/spinner/demo/)
- [rh-stat/demo/](https://deploy-preview-996--red-hat-design-system.netlify.app/elements/statistic/demo/)
- [rh-subnav/demo/](https://deploy-preview-996--red-hat-design-system.netlify.app/elements/subnavigation/demo/)
- [rh-subnav/demo/color-context/](https://deploy-preview-996--red-hat-design-system.netlify.app/elements/subnavigation/demo/color-context/)
- [rh-tabs/demo/](https://deploy-preview-996--red-hat-design-system.netlify.app/elements/tabs/demo/)
- [rh-tabs/demo/color-context/](https://deploy-preview-996--red-hat-design-system.netlify.app/elements/tabs/demo/color-context/)
- [rh-tag/demo/](https://deploy-preview-996--red-hat-design-system.netlify.app/elements/tag/demo/)
- [rh-tooltip/demo/](https://deploy-preview-996--red-hat-design-system.netlify.app/elements/tooltip/demo/)


## Notes to Reviewers
